project-base/src/DataFixtures/Demo/ImageDataFixture.php

```diff
@@ -301,7 +315,7 @@ private function moveFilesFromLocalFilesystemToFilesystem(string $origin, string
     private function truncateImagesFromDb()
     {
         $this->em->getConnection()->executeStatement(
-            'TRUNCATE TABLE ' . self::IMAGES_TABLE_NAME
+            'TRUNCATE TABLE ' . self::IMAGES_TABLE_NAME . ', ' . self::IMAGES_TRANSLATIONS_TABLE_NAME
```

Comment:
shouldn't the CASCADE setting solve this?

Reply:
It probably does, but I do not see problem with declaring it explicitly here.